### PR TITLE
DOCSP-45117-sharded-cluster-behavior

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Adds info on shared cluster behavior and moves sections to the general mongosync behavior page

## STAGING

https://deploy-preview-472--docs-cluster-to-cluster-sync.netlify.app/reference/mongosync/mongosync-behavior/#sharded-clusters
https://deploy-preview-472--docs-cluster-to-cluster-sync.netlify.app/reference/api/start/#behavior

## JIRA

https://jira.mongodb.org/browse/DOCSP-45117

## SELF-REVIEW CHECKLIST

- [ ] Does each file have 3-5 taxonomy facet tags?
  See the [taxonomy tagging instructions](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) and this [example PR](https://github.com/10gen/cloud-docs/pull/5042/files) 
- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## EXTERNAL REVIEW REQUIREMENTS

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/mongosync/mongosync-behavior.txt [hunk 1]
```diff
@@ -68,6 +70,27 @@ Even if documents move between source shards during replication,
...
+``````````````
+
+When ``mongosync`` syncs to a sharded destination cluster, it assigns a
+primary shard to each database by means of a round-robin. 
+
+.. warning::
+
+   Running :dbcommand:`movePrimary` on the source or desintation cluster
+   during migration may result in a fatal error or require you to
+   restart the migration from scratch. For more information, see
```
Comment (jason-price-mongodb) [inline source/reference/mongosync/mongosync-behavior.txt:91]: Consider: "from the start"

File: source/reference/mongosync/mongosync-behavior.txt [hunk 2]
```diff
@@ -68,6 +70,27 @@ Even if documents move between source shards during replication,
...
+
+When ``mongosync`` syncs to a sharded destination cluster, it pre-splits chunks 
+for sharded collections on the destination cluster. For each sharded collection, 
+``mongosync`` creates twice as many chunks as there are shards in the 
+destination cluster. 
+
+Primary Shards
+``````````````
+
+When ``mongosync`` syncs to a sharded destination cluster, it assigns a
```
Comment (jason-price-mongodb) [inline source/reference/mongosync/mongosync-behavior.txt:84]: Consider clarifying what "it assigns" refers to. Cheers.

**Review by jason-price-mongodb — APPROVED** [review]
@ltran-mdb2 LGTM with suggestions. Cheers.

File: source/reference/mongosync/mongosync-behavior.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/reference/mongosync/mongosync-behavior.txt]: Done

File: source/reference/mongosync/mongosync-behavior.txt [hunk 3]
```diff
@@ -68,6 +70,27 @@ Even if documents move between source shards during replication,
 guarantee on the destination cluster. For more information, see 
 :ref:`c2c-sharded-clusters`.
```
Comment (edobranov) [inline source/reference/mongosync/mongosync-behavior.txt:71]: I mentioned in the description of DOCSP-45117 that the balancer / chunk migration stuff no longer applies, so L62- L71 isn't relevant anymore. We can instead 'summarize' the two sub-sections after it by replacing L62 - L71 with:

> {+c2c-product-name+} supports replication between sharded clusters. Individual shards are replicated in parallel from the source cluster to the destination cluster, however the source cluster's sharding configuration is not preserved.

**Review by edobranov — CHANGES_REQUESTED** [review]

File: source/reference/mongosync/mongosync-behavior.txt (hunk 3 above)
Comment (ltran-mdb2) [reply source/reference/mongosync/mongosync-behavior.txt]: Done!

**Review by edobranov — APPROVED** [review]
LGTM, thank you!

**Comment by ltran-mdb2** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|v1.7|[](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/476)|
|✅|v1.8|[](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/477)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

**Pull request merged**